src/main/java/uk/nhs/prm/e2etests/performance/NemsTestEvent.java

```diff
@@ -44,132 +40,77 @@ public static NemsTestEvent nonSuspensionEvent(String nhsNumber, String nemsMess
         return new NemsTestEvent(nemsMessageId, nhsNumber, false);
     }
 
-    public String nemsMessageId() {
-        return nemsMessageId;
-    }
-
-    public String nhsNumber() {
-        return nhsNumber;
-    }
-
-    @Override
-    public void setPhase(LoadPhase phase) {
-        this.phase = phase;
-    }
-
-    @Override
-    public LoadPhase phase() {
-        return phase;
-    }
-
-    public LocalDateTime startedAt() {
-        return startedAt;
-    }
-
-    public boolean isStarted() {
-        return startedAt != null;
-    }
-
-    public boolean isFinished() {
-        return isFinished;
-    }
-
-    public boolean hasWarnings() {
-        return !warnings.isEmpty();
-    }
-
-    public void started(String meshMessageId) {
+    public void start(String meshMessageId) {
         if (meshMessageId == null) {
             addWarning("No mesh message id received. Message may have not arrived in mesh mailbox.");
         }
         this.startedAt = LocalDateTime.now();
         this.meshMessageId = meshMessageId;
     }
 
-    public boolean finished(SqsMessage successMessage) {
-        boolean firstTimeFinisher = false;
-        if (isFinished) {
+    public boolean finish(SqsMessage successMessage) {
+        boolean hasBeenCalled = false;
+        if (isFinished()) {
             addWarning("Warning: Duplicate finisher! finished() but already isFinished");
```

Comment:
We could do to update this as it references a `finished()` method that has been renamed to `finish()` and a field `isFinished` that is now a method.